entrypoint.sh

```diff
@@ -1,42 +1,48 @@
 #!/bin/bash
 
-dataset_f=$1
-participant_f=$2
-out=${3-.}
-jobs=${4-6}
+machine_f=$1
+out=${2-.}
+jobs=${3-6}
 
 getit() {
-    datalad get --dataset $(cut  -d " " -f1 <<< $1) $(cut -d " " -f2 <<< $1)
+    datalad get --dataset $(cut -d " " -f1 <<< $1) $(cut -d " " -f2 <<< $1)
```

Comment:
for readability, how about:
```suggestion
    dataset=$(cut -d " " -f1 <<< $1)
    content=$(cut -d " " -f2 <<< $1)
    datalad get --dataset "$ds" "$content"
```